app/tests/unit/lib/theme/widgets/RevisionStatusWidget.test.tsx

```diff
@@ -118,4 +171,57 @@ describe("The RevisionStatusWidget", () => {
     expect(screen.queryByRole("combobox")).not.toBeInTheDocument();
     expect(screen.queryByText(/update/i)).not.toBeInTheDocument();
   });
+  it("calls commitProjectRevision when the select value is `Applied`", () => {
+    componentTestingHelper.loadQuery();
+    componentTestingHelper.renderComponent();
+    const newStatus = "Applied";
```

Comment:
I find the tests a little harder to follow when we're asserting a variable rather than the actual dropdown text, but no strong feelings